docs/index.rst

```diff
@@ -20,6 +20,18 @@ Pangeo
 A community platform for Big Data geoscience
 ============================================
 
+What is Pangeo?
+^^^^^^^^^^^^^^^
+
+Pangeo is first and foremost a community promoting open reproducible science.
+This community provides instructions, support, and other resources to make
```

Comment:
This community provides documentation, develops software, and deploys computing infrastructure to make

Reply:
My only concern is with the "develops software" part which could give readers the impression that pangeo *is* the group producing the software where (correct me if I'm wrong) it would be more accurate that pangeo's members work with other projects to develop software. I don't feel strongly enough and am not a maintainer so I'm still ok changing it to this.

Reply:
Pangeo basically started out of xarray. Just because there is no package named "pangeo" doesn't mean that software development is not a big part of this effort. Most of the time billed to Pangeo-related grants is software development time. There is no one group that produces the software, but Pangeo has directly participated in a large amount of development in xarray, dask, jupyter, pyviz, intake, etc, whether through direct awards to Anaconda or contributions from folks like @jhamman or @guillaumeeb within academic institutions.